src/BuildScriptGenerator/DotNetCore/DotnetCoreVersionProvider.cs

```diff
@@ -10,6 +10,8 @@ namespace Microsoft.Oryx.BuildScriptGenerator.DotNetCore
 {
     internal class DotNetCoreVersionProvider : IDotNetCoreVersionProvider
```

Comment:
This class might not be required going forward. I did not want to make too many changes on a PR which already has many changes to files.